packages/shared/lib/services/sync/run.service.ts

```diff
@@ -420,11 +461,10 @@ export default class SyncRun {
             }
 
             const startTime = Date.now();
+            const syncStartDate = new Date();
```

Comment:
Could be moved at the very beginning of the method or could be a property of the class directly